signing/osx.cmake

```diff
@@ -8,7 +8,7 @@ add_custom_command(
 
 add_custom_command(
     TARGET signing PRE_BUILD
-    COMMAND codesign -s ${OSX_CODESIGN_IDENTITY} --deep "Ultimaker Cura.app"
+    COMMAND ${CMAKE_SOURCE_DIR}/../sign_cura_osx.sh
```

Comment:
This file doesn't exist. It refers to outside of this repository?!?

Reply:
Yes, CMake would not allow me to create specific commands with regards to signing the .so and .dylib files, so the best solution I could find without bypassing the whole signing process of CMake was to have a file on the build machine that takes care of this.

Reply:
This violates our license and defeats the purpose of having an open source build system. We are legally not allowed to merge this.

Instead, I'd suggest to place this .sh script inside of the repository and call it from CMake in the exact same way.

Reply:
OK, I decided to add the file (changed the 'osx' suffix to 'macOS' as this is the new official name after the launch of Big Sur) in the signing/ dir together with the cura.entitlements file which it references. I was a bit hesitant in the beginning to add the latter since it lists the entitlements we give to our app, but as far as I checked everyone can more or less create one such file using Xcode.

Reply:
> This violates our license. We are legally not allowed to merge this, as far as I can tell. It also defeats the purpose of having an open source build system since nobody could reproduce the build without having that script.
> 
> Instead, I'd suggest to place this .sh script inside of the repository and call it from CMake in the exact same way.

I totally agree. Even though no one would really be able to create a build with `-DSIGN_PACKAGE=ON `anyway, since they would not have the codesign identity that is required for the signing stage ;)